src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueClosureTests.cs

```diff
@@ -125,6 +125,107 @@ void F()
                 Row(5, TableIndex.MethodDef, EditAndContinueOperation.Default));
         }
 
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/43099")]
```

Comment:
I'm working on a fix. Check whether this test can be unskipped when this PR is ready to merge.